packages/react-client/src/ReactFlightClient.js

```diff
@@ -1227,6 +1247,20 @@ function parseModelString(
         }
         return readTemporaryReference(temporaryReferences, reference);
       }
+      case '*': {
```

Comment:
Doesn’t seem worth taking two characters for this. Could just be “$L” just like “$@“ is an ID less promise.

Reply:
Just figure lazy wrappers are so common it wouldn’t be worth this check when it’ll only be produced (for now) by pre-renders. It’s a little cost for a common path. We do have a decent number of characters remaining and seemingly not many plans for how we would consume them. But I’ll defer to you I don’t feel strongly about it

Reply:
And by this logic I’d argue we find another character to use for &@ too :)

Reply:
Talked offline but we actually don't have a lot of characters left. At least not with the current (somewhat arbitrary constraints).

- It has to be ASCII 7-bit to be encoded as a single utf-8 byte.
- It can't be any of the characters that require escaping in JSON like newline, tab, quotation mark for the same reason.
- Currently we reserve the hexadecimal lower case `a-f` and `1-9` to save a byte on direct references.
- It should ideally be the same character in the Flight Server/Client and the Flight Reply Server/Client for debugging consistency but also to ensure that we're to add any feature to either direction. E.g. maybe hints are useful for Reply. We should also support streaming replies in the future so some more of these might be useful. Temporary references is a new example that goes both direction.
 
So it's really a union of these and the Reply set plus maybe some of the row tags.

https://github.com/facebook/react/blob/main/packages/react-server/src/ReactFlightReplyServer.js#L917-L1083